src/2ls/2ls_parse_options.h

```diff
@@ -66,7 +66,7 @@ class optionst;
   "(graphml-witness):(json-cex):" \
   "(no-spurious-check)(stop-on-fail)" \
   "(competition-mode)(slice)(no-propagation)(independent-properties)" \
-  "(no-unwinding-assertions)"
+  "(constant-propagation)(no-unwinding-assertions)"
```

Comment:
:pick: put each declaration on a separate line to reduce merge conflicts in future.